.travis.yml

```diff
@@ -32,8 +33,27 @@ jobs:
       before_install: dev/before_install
       before_script: dev/before
     - stage: test
+      name: Linux + Oracle JDK 9
+      os: linux
+      dist: trusty
+      sudo: required
+      jdk: oraclejdk9
+      install: true
+      script: dev/main
+      before_install: dev/before_install
+      before_script: dev/before
+    - stage: test
+      name: macOS + Oracle JDK 8
+      os: osx
+      osx_image: xcode9.3
+      install: true
+      script: dev/main
+      before_install: dev/before_install
+      before_script: dev/before
+    - stage: test
+      name: macOS + Oracle JDK 10
```

Comment:
why not linux and jdk 10?

Reply:
see https://docs.travis-ci.com/user/reference/trusty/#jvm-clojure-groovy-java-scala-images

Reply:
for macOS, I wanted to see if JDK 10 works at all, it does not have to be there.

Reply:
aha, ok not supported.